src/main/java/com/google/api/codegen/util/java/JavaTypeTable.java

```diff
@@ -174,4 +175,62 @@ private boolean isImplicitImport(String name) {
     implicitImports.put(name, yes);
     return yes;
   }
+
+  public static final ImmutableSet<String> RESERVED_IDENTIFIER_SET =
```

Comment:
I'm not sure, I guess they don't have the same problems with conflicts?
@garrettjonesgoogle